synapse_pay_rest/tests/models/document_tests.py

```diff
@@ -36,9 +36,7 @@ def setUp(self):
 
     def test_physical_document_with_bytestream(self):
         type = 'GOVT_ID'
-        file_path = ('/Users/Synapsepay/Documents/libraries/'
-                     'SynapsePayRest-Python/synapse_pay_rest/'
-                     'tests/fixtures/test.png')
+        file_path = input("Input file test file full path")
```

Comment:
I can see the issue with using an absolute path here, but this approach interrupts the test midway. It's important to be able to let the test run and get the results without having to babysit it. Is there some way to rewrite this with a relative path?